react_app/src/App.js

```diff
@@ -1,27 +1,30 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import EmbeddingForm from "./Form";
+import "./App.scss";
 
-const App = () => {
+const DATASET_CREATE_STRING = "Create a dataset";
+const items = [
+  {
+    index: 0,
+    title: "Model",
+    values: ["DeLF", "other"]
```

Comment:
Maybe you can add "other" later when you really support a different option?

Reply:
Fixed